<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MellowPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mellowplayer.qrc">
    <normaloff>:/icons/mellowplayer.png</normaloff>:/icons/mellowplayer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageHome">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/icons/mellowplayer.png&quot; width=&quot;128&quot; height=&quot;128&quot;/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Welcome to MellowPlayer!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;Please, select a music streaming service.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonPreferences">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Preferences</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-system">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonQuit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Quit</string>
            </property>
            <property name="icon">
             <iconset theme="application-exit">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select a music streaming service</string>
            </property>
            <property name="icon">
             <iconset theme="system-run">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageWeb">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WebView" name="webView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Applicatio&amp;n</string>
    </property>
    <addaction name="actionSelect_service"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playe&amp;r</string>
    </property>
    <addaction name="actionPlayPause"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_to_favorites"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout_MellowPlayer"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionWiki"/>
    <addaction name="actionReport_a_bug"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShow_menu"/>
    <addaction name="actionShow_toolbar"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navi&amp;gation</string>
    </property>
    <addaction name="actionHome"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionSelect_service"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_to_favorites"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlayPause"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionSelect_service">
   <property name="icon">
    <iconset theme="system-run" resource="mellowplayer.qrc">
     <normaloff>:/icons/system-run.png</normaloff>:/icons/system-run.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Select a music streaming service</string>
   </property>
   <property name="toolTip">
    <string>Select a music streaming service</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system" resource="mellowplayer.qrc">
     <normaloff>:/icons/preferences-system.png</normaloff>:/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset theme="media-playback-start" resource="mellowplayer.qrc">
     <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Media Play</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="media-seek-forward" resource="mellowplayer.qrc">
     <normaloff>:/icons/media-seek-forward.png</normaloff>:/icons/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Media Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="media-seek-backward" resource="mellowplayer.qrc">
     <normaloff>:/icons/media-seek-backward.png</normaloff>:/icons/media-seek-backward.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;revious</string>
   </property>
   <property name="shortcut">
    <string>Media Previous</string>
   </property>
  </action>
  <action name="actionAbout_MellowPlayer">
   <property name="icon">
    <iconset theme="help-about" resource="mellowplayer.qrc">
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About MellowPlayer</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReport_a_bug">
   <property name="icon">
    <iconset theme="tools-report-bug" resource="mellowplayer.qrc">
     <normaloff>:/icons/tools-report-bug.png</normaloff>:/icons/tools-report-bug.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Report a bug</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit" resource="mellowplayer.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRestoreWindow">
   <property name="icon">
    <iconset theme="view-restore" resource="mellowplayer.qrc">
     <normaloff>:/icons/view-restore.png</normaloff>:/icons/view-restore.png</iconset>
   </property>
   <property name="text">
    <string>Restore window</string>
   </property>
   <property name="toolTip">
    <string>Restore the window</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/about-qt.png</normaloff>:/icons/about-qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAdd_to_favorites">
   <property name="icon">
    <iconset theme="emblem-favorite">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Add to favorites</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="icon">
    <iconset theme="help-contents" resource="mellowplayer.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Wiki</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="go-home" resource="mellowplayer.qrc">
     <normaloff>:/icons/go-home.png</normaloff>:/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
  </action>
  <action name="actionShow_menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShow_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;how toolbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-fullscreen">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>webview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonSelect</tabstop>
  <tabstop>pushButtonPreferences</tabstop>
  <tabstop>pushButtonQuit</tabstop>
 </tabstops>
 <resources>
  <include location="mellowplayer.qrc"/>
 </resources>
 <connections/>
</ui>
